test/catch2/live_restore/unit/test_live_restore_bitmap_encode_decode.cpp

```diff
@@ -52,29 +47,42 @@ TEST_CASE("Encode various bitmaps", "[live_restore_bitmap],[live_restore_bitmap_
     WT_ITEM buf;
     WT_CLEAR(buf);
 
-    REQUIRE(__wt_rwlock_init(session, &lr_fh.lock) == 0);
-
+    live_restore_test_env env;
     for (const auto &test : test_bitmaps) {
-        lr_fh.bitmap = test.bitmap;
-        lr_fh.nbits = test.nbits;
-        // We need to have a non NULL pointer here for the encoding to take place.
-        lr_fh.source = reinterpret_cast<WT_FILE_HANDLE *>(0xab);
-        __wt_readlock(session, &lr_fh.lock);
-        REQUIRE(__ut_live_restore_encode_bitmap(session, &lr_fh, &buf) == 0);
-        __wt_readunlock(session, &lr_fh.lock);
+        size_t filesize = test.nbits == 0 ? 4096 : 4096 * test.nbits;
```

Comment:
nit: local variables to stop calling `env.dest_file_path` everywhere 
```
        std::string dest_file = env.dest_file_path("file");
        std::string source_file = env.source_file_path("file");
```

Reply:
e9a8200543